### ISSUE TITLE
Support for React 16.0.0^

### ISSUE THREAD

**Issue by advance512** [lorensr/segmented-control#5]
```
    "react": "^15.2.1",
```

**Comment by advance512** [comment]
Also:

```
Warning: SegmentedControl: React.createClass is deprecated and will be removed in version 16. Use plain JavaScript classes instead. If you're not yet ready to migrate, create-react-class is available on npm as a drop-in replacement.
```

**Comment by huynguyenh** [comment]
@advance512: i did create a pull request #9 
Please give it a look

**Comment by lorensr** [comment]
Fixed in `v0.1.9` by #10
